packages/service-library/src/servicelib/rabbitmq_errors.py

```diff
@@ -0,0 +1,38 @@
+from typing import Final
+
+from pydantic.errors import PydanticErrorMixin
+
+_ERROR_PREFIX: Final[str] = "rabbitmq.robust_rpc."
```

Comment:
Q: why double "a.b" and not simply "rabbitmq_error" like the module name?

Reply:
good catch, this got refactored a lot.